examples/dsa_hisa/README.md

```
@@ -0,0 +1,200 @@
+    blocked_kv_fp8,           # [Nb, D]   fp8     (from step 1.1)
+    blocked_kv_scale,         # [Nb]      f32     (from step 1.1)
+    weights_f32,              # [M, H]    f32
+    cu_seqlen_blocked_ks,     # [M] int32 — per-query start in pool-block coords
+    cu_seqlen_blocked_ke,     # [M] int32 — per-query end   in pool-block coords
+)
+# block_k_score: [M, Nb] f32
+```
+
+**What it does**: for each query `m` and each pool block `n` in
+`[cu_seqlen_blocked_ks[m], cu_seqlen_blocked_ke[m])`,
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)